massgen/tests/test_bootstrap_criteria.py

```
@@ -443,6 +455,601 @@ def test_subagent_mode_still_propagates_seeded_entries(self):
+        orch.coordination_tracker = SimpleNamespace(
+            answers_by_agent={
+                "a1": [SimpleNamespace(content="Answer from agent a1", agent_id="a1")],
+                "a2": [SimpleNamespace(content="Answer from agent a2", agent_id="a2")],
+            },
+        )
+        orch.current_task = "Design a logo"
+        orch.session_id = "test-session"
+        # Workspace + log dir don't matter when SubagentManager is mocked, but
+        # the method may attempt to resolve them; stub minimal attributes.
+        orch.agents = {
+            "a1": SimpleNamespace(backend=SimpleNamespace(filesystem_manager=SimpleNamespace(cwd="/tmp/test_ws"))),
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/test_ws"

(S108)